amf-aml/shared/src/main/scala/amf/client/model/domain/DialectDomainElement.scala

```diff
@@ -38,20 +38,21 @@ case class DialectDomainElement(override private[amf] val _internal: InternalDia
 
   def includeName(): String = _internal.includeName
 
-  def setObjectProperty(propertyId: String, value: DialectDomainElement): InternalDialectDomainElement = {
```

Comment:
Same change in all of them. The Internal was returned instead of the client one.